web/packages/teleport/src/Discover/Database/CreateDatabase/CreateDatabase.story.tsx

```diff
@@ -27,12 +27,18 @@ export default {
   title: 'Teleport/Discover/Database/CreateDatabase',
 };
 
-export const InitSelfHosted = () => (
+export const InitSelfHostedPostgres = () => (
   <MemoryRouter>
     <CreateDatabaseView {...props} />
```

Comment:
nit: pass `dbEngines={DatabaseEngine.PostgreSQL}` here and changes the props default to `dbEngine: null`?

I know it won't actually change anything but it might make the code a little easier to follow. Not sure if this will cause problems though, I'm no JS expert 😅

Reply:
great comment and makes sense :), but i left as is though (but i clarified a story name), b/c these are just written sample stories to visually check out how it gets rendered on the web so the code itself doesn't matter too much (and it gets repetitive when writing different states)